src/main/java/frc/trigon/robot/subsystems/climber/ClimberCommands.java

```diff
@@ -0,0 +1,67 @@
+package frc.trigon.robot.subsystems.climber;
+
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.StartEndCommand;
+import frc.trigon.robot.RobotContainer;
+import org.trigon.commands.NetworkTablesCommand;
+
+public class ClimberCommands {
+    public static Command getDebuggingCommand() {
+        return new NetworkTablesCommand(
+                (positionMeters, affectedByRobotWeight) -> ClimberCommands.getSetTargetPositionCommand(positionMeters, positionMeters, affectedByRobotWeight == 1),
+                false,
+                "Debugging/TargetDebuggingClimberPositionMeters",
+                "Debugging/TargetDebuggingClimberPositionAffectedByRobotWeight"
+        );
+    }
+
+    public static Command getSetTargetPositionCommand(double targetRightPositionMeters, double targetLeftPositionMeters, boolean affectedByRobotWeight) {
+        return new StartEndCommand(
+                () -> RobotContainer.CLIMBER.setTargetPosition(targetRightPositionMeters, targetLeftPositionMeters, affectedByRobotWeight),
+                RobotContainer.CLIMBER::stop,
+                RobotContainer.CLIMBER
+        );
+    }
+
+    public static Command getSetTargetStateCommand(ClimberConstants.ClimberState targetState) {
+        return new StartEndCommand(
+                () -> RobotContainer.CLIMBER.setTargetState(targetState),
+                RobotContainer.CLIMBER::stop,
+                RobotContainer.CLIMBER
+        );
+    }
+
+    public static Command getSetTargetVoltageCommand(double targetVoltage) {
+        return new StartEndCommand(
+                () -> RobotContainer.CLIMBER.drive(Units.Volt.of(targetVoltage)),
+                RobotContainer.CLIMBER::stop,
+                RobotContainer.CLIMBER
+        );
+    }
+
+    public static Command getSetRightMotorTargetVoltageCommand(double targetVoltage) {
+        return new StartEndCommand(
+                () -> RobotContainer.CLIMBER.rightMotorDrive(Units.Volt.of(targetVoltage)),
+                RobotContainer.CLIMBER::stop,
+                RobotContainer.CLIMBER
+        );
+    }
+
+    public static Command getSetLeftMotorTargetVoltageCommand(double targetVoltage) {
+        return new StartEndCommand(
+                () -> RobotContainer.CLIMBER.leftMotorDrive(Units.Volt.of(targetVoltage)),
+                RobotContainer.CLIMBER::stop,
+                RobotContainer.CLIMBER
+        );
+    }
```

Comment:
Never create a command that only does one thing for half of the subsystem. This will create weird requirements. Have one command with two params.